backend/app/assets/stylesheets/spree/backend/globals/_fonts.scss

```diff
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: 'Inter';
+  font-style:  normal;
+  font-weight: 400;
+  src: url("Inter-Regular.woff2") format("woff2"),
+       url("Inter-Regular.woff") format("woff");
+}
+
+@font-face {
+  font-family: 'Inter';
+  font-style:  normal;
+  font-weight: 600;
+  src: url("Inter-SemiBold.woff2") format("woff2"),
+       url("Inter-SemiBold.woff") format("woff");
+}
```

Comment:
Final newline missing

Reply:
Thanks @elia !
Actually few weeks ago I played with Atom settings, and from that moment something went weird. I'm thinking about reset settings to my last backup.